modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -40,72 +40,172 @@ or the xref:{version-server}@server:manage:manage-settings/install-sample-bucket
 
-
 == Step by Step
 
-Now you have the Go client installed, open your preferred text editor and try out the following:
+At this point we want to transition from the terminal to your code editor of choice.
+
+Let's now create an empty file named `main.go` and walk through adding code step-by-step:
+
+* Connect to a single node cluster, bucket, and collection;
+* Add and retrieve a new document;
+* Look up (SQL-type query) the new document by attribute value.
+
+=== Prerequisites
+
+Here are all of the imports that you will need to run the sample code.
+
+[source,golang]
+----
+include::devguide:example$go/cloud.go[tags=imports,indent=0]
+----
+
+Now, create an empty `main()` function.
 
 [source,golang,indent=0]
 ----
-include::example$startusing.go[tag=connect]
+func main() {
+	// add code here...
+}
 ----
 
-If you are connecting to https://docs.couchbase.com/cloud/index.html[Couchbase Capella] rather than a local Couchbase Server, then also refer to the <<cloud-connections, Cloud section>>, below.
+We will update this function as we go along the steps in this guide.
 
-Couchbase uses xref:7.0@server:learn:security/roles.adoc[Role Based Access Control (RBAC)] to control access to resources.
-Here we will use the _Full Admin_ role created during installation of the Couchbase Data Platform.
-For production client code, you will want to use more appropriate, restrictive settings -- but here we want to get you up and running quickly.
-If you're developing client code on the same VM or machine as the Couchbase Server, your URI can be _localhost_.
+=== Connection
 
-To access the KV (Key/Value) API or to query views, you need to open a `Bucket`:
+The basic connection details that you'll need are given below -- for more background information, refer to the xref:howtos:managing-connections.adoc#connection-strings[Managing Connections page].
 
-[source,golang,indent=0]
+In the `main()` function, add the following:
+
+[tabs]
+====
+Couchbase Capella::
++
+--
+IMPORTANT: Couchbase Capella requires mandatory use of TLS (Transport Layer Security).
+As of Go SDK version 2.5.0, the standard certificate required to connect to a Capella cluster is automatically included with no additional configuration.
+
+[source,golang]
 ----
-include::example$startusing.go[tag=bucket]
+include::devguide:example$go/cloud.go[tag=connect,indent=0]
 ----
 
-If you installed the travel sample data bucket, replace _bucketName_ with _travel-sample_.
+Couchbase Capella uses xref:cloud:organizations:organization-projects-overview.adoc[Roles] to control user access to database resources.
+For the purposes of this example, we can use the _Organization Owner_ role automatically assigned to a user account during installation of the Capella cluster.
+This role grants full access to a Capella organization, including full data access to all projects and clusters.
+In a production scenario, you may wish to set up users with more granular access roles. 
```

Comment:
If we say will instead of may, makes it clearer that it's required as good practice.